port/pool.go

```diff
@@ -0,0 +1,46 @@
+package port
+
+import (
+	log "github.com/cihub/seelog"
+	"math/rand"
+	"time"
+)
+
+type Port struct {
+	Number int
+}
+
+func (p *Port) Release() {
```

Comment:
All exported methods should have documentation.

Reply:
If you acquire through pool, you should return through pool too. Imho Port struct is not very needed here at all.